docs/en_US/Release.rst

```diff
@@ -5,6 +5,69 @@
 Change Log
 ==========
 
+Release 2.1 - 3/8/2021
+-----------------------
+
+Major updates
+^^^^^^^^^^^^^
+
+Neural architecture search
+""""""""""""""""""""""""""
+
+* Improve NAS 2.0 (Retiarii) Framework (Alpha release)
+
+  * `Feature roadmap <https://github.com/microsoft/nni/issues/3301>`__
+  * `Related issues and pull requests <https://github.com/microsoft/nni/issues?q=label%3Aretiarii-v2.1>`__
+  * Documentation improvement (#3404 #3432)
+
+Training service
+""""""""""""""""
+
+* Support shared storage for reuse mode (#3354)
+* Support local platform for Windows in hybrid training service (#3353)
+* Remove PAIYarn training service (#3327)
+* Add recently-idle environment scheduler in reuse mode (#3375)
+* Rename ``preCommand`` to ``pythonPath`` and change command lines as serial in remote training service (#3284 #3410)
+
+nnictl & nni.experiment
+"""""""""""""""""""""""
+
+* Migrate nnicli to new Python API ``nni.experiment`` (#3334)
+* Support resuming an experiment launched from Python by nnictl (#3309)
+* Align ``nni.experiment`` tuner behavior with nnictl (#3419)
```

Comment:
-> Refactor the way of specifying tuner in experiment Python API (``nni.experiment``), more aligned with nnictl